_plugins/content_typography.rb

```diff
@@ -0,0 +1,12 @@
+module Kramdown
+  module Parser
+    class Kramdown
+      prepend(Module.new do
+        def add_link(el, *args)
+          el.attr['class'] = [*el.attr['class'], 'usa-link'].join(' ') if el.type == :a
+          super(el, *args)
```

Comment:
Could serve as motivation to bump to Ruby >=2.7 😄 

```suggestion
        def add_link(el, ...)
          el.attr['class'] = [*el.attr['class'], 'usa-link'].join(' ') if el.type == :a
          super(el, ...)
```